pages/apply/overview.tsx

```diff
@@ -34,6 +37,19 @@ const ApplicationPersonsOverview = (): JSX.Element => {
       .flat()
   );
 
+  const mainApplicant = useAppSelector(
+    (store) => store.application.mainApplicant
+  );
+
+  const [isEligible] = useMemo(
```

Comment:
Check status of the application instead, requires updating the application via API on disqualification.